version.go

```diff
@@ -1,7 +0,0 @@
-package skopeo
-
-// Version is a version of thils build.
-const Version = "0.1.12-dev"
```

Comment:
Assume that either the current version of the code has a bug which makes the signature invalid but still with a valid structure (say, computing a hash over UCS-4 representation instead of a raw `[]bytes`, using a non-canonical Docker reference, or incorrectly stripping a trailing `'\n'` _if there is one_). In the last example it may very well be a bug we notice only after a few months/years, with both correct and invalid signatures in the wild, both sharing the same formal version number.

In such cases, having the precise version number of the creator allows us to deploy targeted workarounds (when verifying a signature made by a known buggy version, compute the hash in the same way the buggy version does).